src/main/java/cloud/fogbow/ras/core/plugins/interoperability/aws/attachment/v2/AwsV2AttachmentPlugin.java

```diff
@@ -59,87 +55,87 @@ public boolean hasFailed(String instanceState) {
 
 	@Override
 	public String requestInstance(AttachmentOrder attachmentOrder, AwsV2User cloudUser) throws FogbowException {
-		LOGGER.info(String.format(Messages.Info.REQUESTING_INSTANCE, cloudUser.getToken()));
+		LOGGER.info(String.format(Messages.Info.REQUESTING_INSTANCE_FROM_PROVIDER));
```

Comment:
should we make this class more homogeneous? for example, we have a sort of pattern here: 1) create client; 2) create request; 2) perform the request. However, part 3 varies. 1) it can be made in the body of the method; 2) it can be made in a private method; 3) it can be made in the AwsV2CloudUtil class.
Is it worth uniformizing it?

Reply:
Actually I think the request varies a lot from one operation to another. This sort of pattern has been made with the goal of uniforming it... For example, how would be the code of a private method that create the client and the request of requestInstance and deleteInstance operations?. (I don't know if that's what you mean.)

Reply:
no. I meant: create a private `doDetachVolume` method and call it in the `deleteInstance`